### PR TITLE
feature/bash

### PR DESCRIPTION
**Pull request by JordiRZ**

### CODE REVIEW

**Comment by manuelbarzi** [comment]
ADD the README.md!!! .P

**Comment by heberjs** [comment]
Make your code readable and organized.

**Comment by JordiRZ** [comment]
> Make your code readable and organized.

I dont think so

File: staff/jordi-relano/bash/README.me [hunk 1]
```diff
@@ -0,0 +1,49 @@
+```bash										
+las comillas invertidas generan un espacio cuando tabulamos
```
Comment (Barrufat) [inline staff/jordi-relano/bash/README.me]: Organize and follow the example of Manu's file https://drive.google.com/drive/folders/1zIyeQgKR3nIPAuaQ5jA3Av5h9MHprHxd and remember to write everything in english 🧐

File: staff/jordi-relano/bash/README.me [hunk 2]
```diff
@@ -0,0 +1,53 @@
+BASH
+
+#```
```
Comment (manuelbarzi) [inline staff/jordi-relano/bash/README.me]: correct this "popourri" .P

File: staff/jordi-relano/bash/README.me [hunk 3]
```diff
@@ -0,0 +1,80 @@
+#BASH
+
+```sh
+
+##pwd (Command to see where you are located)
```
Comment (Barrufat) [inline staff/jordi-relano/bash/README.me:5]: You should close each command definition with the back ticks closure ```.
Each section must be defined inside he back ticks structure.

File: staff/jordi-relano/bash/README.me [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+```sh
+$pwd										
+																				
+##ls --Command to view folder directory list (all folders)--									
+
```
Comment (Barrufat) [inline staff/jordi-relano/bash/README.me:10]: You should close each command definition with the back ticks closure ```.

**Pull request closed without merging**